### PR TITLE
Disable annotation commands by default

### PR DESCRIPTION
**Pull request by StoneT2000**

### CODE REVIEW

**Comment by StoneT2000** [comment]
@bovard this should disable annotation commands by default

Would prefer if we could have this enabled by default but only if the servers can pass in specific configs. (Which would include setting loglevel=0, annotations=False)

File: kaggle_environments/envs/lux_ai_2021/lux_ai_2021.py [hunk 1]
```diff
@@ -56,8 +57,13 @@ def interpreter(state, env):
         if "loglevel" in env.configuration:
             loglevel = env.configuration["loglevel"]
         else:
-            loglevel = 2 # warnings, 1: errors, 0: none
+            loglevel = 0 # warnings, 1: errors, 0: none
```
Comment (bovard) [inline kaggle_environments/envs/lux_ai_2021/lux_ai_2021.py:60]: @StoneT2000 logging off by default?

Comment (bovard) [reply kaggle_environments/envs/lux_ai_2021/lux_ai_2021.py:60]: ah, looks like this is system logs, not player logs

**Review by bovard — APPROVED** [review]
LGTM

**Pull request merged**